Release notes 2.14.6 (#11234)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.14.6

> 

Reviewer: https://github.com/PrefectHQ/prefect/pull/11230

The UI enhancement!

│                                                                                                          │
│         - Create GCP credentials block my--pool-push-pool-credentials to store the service account key   │
│                                                                                                          │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Proceed with infrastructure provisioning? [y/n]: y
Activating Cloud Run API
Creating service account
Assigning roles to service account
Creating service account key
Creating GCP credentials block
Provisioning Infrastructure ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Infrastructure successfully provisioned!
Created work pool 'my-pool'!
```

If you have yet to try using a Cloud Run run push pool, now is a great time!

If you use another cloud provider, don't fret; we will add support for ECS and Azure Container Instances push work pools in future releases!

See the following pull request for implementation details:
— https://github.com/PrefectHQ/prefect/pull/11204

### Enhancements

> - Add ability to search for block documents by name — https://github.com/PrefectHQ/prefect/pull/11212

Reviewer: https://github.com/PrefectHQ/prefect/pull/11212

> - Add pagination to the blocks page — https://github.com/PrefectHQ/prefect/pull/11214

Reviewer: Add pagination to the Blocks page in the Prefect UI for viewing/filtering more than 200 blocks

> - Include concurrency controls in `prefect-client` — https://github.com/PrefectHQ/prefect/pull/11227

Reviewer: @urimandujano is this noteworthy enough for a highlight?

> - Add display to how the next run for a deployment — https://github.com/PrefectHQ/prefect/pull/11230

Reviewer: Move to highlight section 🎉

### Fixes
- Fix SQLite migration to work with older SQLite versions — https://github.com/PrefectHQ/prefect/pull/11215

> - Fix subflow filters and persist to URL in flow run search — https://github.com/PrefectHQ/prefect/pull/11218

Reviewer: Fix Subflow Runs tab filters and persist to URL in the Flow Runs page of the Prefect UI

### Documentation
- Improve formatting in deployment guides — https://github.com/PrefectHQ/prefect/pull/11217

> - Add instructions for turning off the flow run logger in the unit testing guide — https://github.com/PrefectHQ/prefect/pull/11223

Reviewer: ```suggestion
- Add instructions for turning off the flow run logger to the unit testing guide — https://github.com/PrefectHQ/prefect/pull/11223
```

# Prefect Release Notes

## Release 2.14.6

### View the next run for a deployment at a glance

> You can now see the next run for a deployment on the details page for a deployment. Upcoming runs are now located in a dedicated tab, making the most relevant running and completed flow runs more visible.

Reviewer 2: ```suggestion
You can now see the next run for a deployment on the details page for a deployment. Upcoming runs are now located in a dedicated tab, making the most relevant running and completed flow runs more apparent.
```

> You can now see the next run for a deployment on the details page for a deployment. Upcoming runs are now located in a dedicated tab, making the most relevant running and completed flow runs more visible.

Reviewer: ```suggestion
You can now see the next run for a deployment on the Runs tab of the Deployments page in the Prefect UI! Upcoming runs are now located in a dedicated tab, making the most relevant running and completed flow runs more visible.
```

> Click below out this video to see it in action!

Author: ```suggestion
Click below to see it in action!
```
